python/ray/autoscaler/_private/commands.py

```diff
@@ -122,7 +128,7 @@ def create_or_update_cluster(config_file: str,
                              yes: bool,
                              override_cluster_name: Optional[str] = None,
                              no_config_cache: bool = False,
-                             redirect_command_output: bool = False,
+                             redirect_command_output: Optional[bool] = False,
```

Comment:
I think this can remain a `bool`?

Reply:
It looks like it could remain a `bool`, but right now that's not consistent with the code.
Currently, it is a passed a `None` value from the corresponding function in [sdk.py](https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/sdk.py#L40): 
Also, the [current logic](https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/commands.py#L131) is set up to handle `None` as default value.

It looks like the `None`-handling logic was inserted for a reason. @maximsmol probably knows.